### ISSUE TITLE
Use index tooltip on the main statistics page

### ISSUE THREAD

**Issue by mojca** [macports/macports-webapp#306]
The statistic graphs for individual ports (`app/port/templates/port/port_stats.htm`) mostly use
```js
tooltips: {
    mode: 'index',
}
```
which allows seeing values for the full column at once.

Knowing the numbers on the main statistics page however requires hovering over super tiny sub-pixel high bars which is almost a mission-impossible.

It might make sense to use index tooltips on the main statistics graphs as well.

It might also make sense to slightly strip the xcode and clt version numbers, but that's likely for another ticket.

**Comment by Dhruv-Sachdev1313** [comment]
Hey! @mojca, I see this issue was opened 6 months back. I would love to work on this if this still needs to be fixed!

**Comment by mojca** [comment]
@Dhruv-Sachdev1313: awesome! Sure, it's still a desirable fix and I'm not aware of anyone working on this. Feel free to go ahead and ask if something is unclear.

**Comment by Dhruv-Sachdev1313** [comment]
@mojca by main statistics page you mean the macPorts Version bar chart on https://ports.macports.org/statistics/ page right?
If this is it, then it seems a very small fix, I can do it right away!

**Comment by arjunsalyan** [comment]
@Dhruv-Sachdev1313 Yes, https://ports.macports.org/statistics/ is referred to as the main statistics page here.
